aquadoggo/src/db/stores/log.rs

```diff
@@ -11,7 +11,10 @@ use p2panda_rs::storage_provider::errors::LogStorageError;
 use p2panda_rs::storage_provider::traits::{AsStorageLog, LogStore};
 
 use crate::db::provider::SqlStorage;
-
+/// Represents a stored log in a way that is conducive to implementing storage traits.
+///
+/// In the case of `LogRow`, the traits could have been implemented directly on it but this
+/// structure was chosen to align with the way the other stores are structured.
```

Comment:
@sandreae is this correct?

Reply:
Fun fact: we actually never retrieve a log from storage.... only the log id... so that's even more redundancy :rofl:

Reply:
I don't know how best to write that, saying that the later is "laid out like the trait methods params"  seems off to me. The reason for each struct is:

1) `sqlx` can infer the type of retrieved returned query if there is a struct with matching rows, that's `LogRow`
2) Our `StorageProvider` trait requires each sub-store to also implement a trait relating to each stores primary data type, that's `StorageLog`

In this case we could make `1)` and `2)` the same struct, but currently choose not to for consistency across storage modules.